content/en/docs/reference/labels-annotations-taints/_index.md

```diff
@@ -92,6 +92,55 @@ Common forms of values include:
+
+### applyset.k8s.io/contains-group-resources
+
+Example: `applyset.k8s.io/contains-group-resources: "certificates.cert-manager.io,configmaps,deployments.apps,secrets,services"`
+
+Used on: Objects being used as ApplySet parents. As of Kubernetes version {{< skew currentVersion >}}, this may be Secrets, ConfigMaps, or custom resources if the Custom Resource Definition defining them has the `applyset.k8s.io/is-parent-type` label.
+
+Part of the [ApplySet specification](https://github.com/kubernetes/enhancements/tree/master/keps/sig-cli/3659-kubectl-apply-prune#design-details-applyset-specification). This annotation is applied to the parent object used to track an ApplySet to optimize listing of ApplySet member objects. It is optional in the ApplySet specification, as tools can perform discovery or use a different optimization. However, as of Kubernetes version {{< skew currentVersion >}}, it is required by kubectl. When present, the value of this annotation must be a comma separated list of the group-kinds, in the fully-qualified name format, i.e. `<resource>.<group>`.
+
+
+### applyset.k8s.io/id
+
+Example: `applyset.k8s.io/id: "applyset-0eFHV8ySqp7XoShsGvyWFQD3s96yqwHmzc4e0HR1dsY-v1"`
+
+Used on: Objects being used as ApplySet parents. As of Kubernetes version {{< skew currentVersion >}}, this may be Secrets, ConfigMaps, or custom resources if the Custom Resource Definition defining them has the `applyset.k8s.io/is-parent-type` label.
+
+Part of the [ApplySet specification](https://github.com/kubernetes/enhancements/tree/master/keps/sig-cli/3659-kubectl-apply-prune#design-details-applyset-specification). This label is what makes an object an ApplySet parent object. Its value is the unique ID of the ApplySet, which is derived from the identity of the parent object itself. This ID MUST be the base64 encoding (using the URL safe encoding of RFC4648) of the hash of the group-kind-name-namespace of the object it is on, in the form: `<base64(sha256(<name>.<namespace>.<kind>.<group>))>`.
+
+### applyset.k8s.io/is-parent-type
+
+Example: `applyset.k8s.io/is-parent-type: "true"`
+
+Used on: Custom Resource Definition (CRD)
+
+Part of the [ApplySet specification](https://github.com/kubernetes/enhancements/tree/master/keps/sig-cli/3659-kubectl-apply-prune#design-details-applyset-specification). This label can be set on a CRD to identify the custom resource type it defines (not the CRD itself) as an allowed parent for an ApplySet. The value of the label MUST be `"true"`.
+
+### applyset.k8s.io/part-of
+
+Example: `applyset.k8s.io/part-of: "applyset-0eFHV8ySqp7XoShsGvyWFQD3s96yqwHmzc4e0HR1dsY-v1"`
+
+Used on: All objects.
+
+Part of the [ApplySet specification](https://github.com/kubernetes/enhancements/tree/master/keps/sig-cli/3659-kubectl-apply-prune#design-details-applyset-specification). This label is what makes an object a member of an ApplySet. The value of the label MUST match the value of the `applyset.k8s.io/id` label on the parent object.
```

Comment:
We should link to the ApplySet docs, not the KEP.

Also, use `**must**` not `MUST` - this is Markdown and we can use bold, italics etc.